packages/esm-patient-biometrics-app/src/biometrics/biometrics-overview.scss

```diff
@@ -15,23 +15,43 @@
   background-color: $ui-background;
 }
 
-.biometricsHeaderContainer > h4:after {
+.biometricsHeaderContainer>h4:after {
   content: "";
   display: block;
   width: 2rem;
   padding-top: 0.188rem;
   border-bottom: 0.375rem solid $brand-teal-01;
 }
 
+.activeButton {
+  background-color:$inverse-link;opacity: 0.7;
+  border: solid $inverse-link 2px;
+  color: $color-blue-60-2;
+}
+
+.toggle {
+  border: 1px solid $inverse-link;opacity: 0.7;
+}
+
 .toggleButtons {
   width: fit-content;
   margin: 0 $spacing-03;
+  padding-left: 49.875rem;
+  height: 2.5rem;
+  border-radius: 4px;
 }
 
 .toggle:first-of-type {
   border-radius: $spacing-02 0 0 $spacing-02;
 }
 
-.toggle:last-of-type {
```

Comment:
Hm there are some changes in this (not only this line, also others) where the code got "uglier". Any reason for that?

Reply:
What do you mean by "uglier"?

Reply:
Removal of spaces for readability.

Please compare the lines your self...

Reply:
Ok, please let me know if this file looks better now, thanks